README.md

```diff
@@ -8,8 +8,19 @@ This crate provides both a library to interact with iMessage data as well as a b
 
 The `imessage-exporter` binary exports iMessage data to `txt` or `html` formats. It can also run diagnostics to find problems with the iMessage database.
 
+### Installation
+
+#### From source
 Installation instructions for the binary are located [here](imessage-exporter/README.md).
 
+
+#### Using Homebrew (macOS)
```

Comment:
This section belongs here: https://github.com/ReagentX/imessage-exporter/tree/develop/imessage-exporter#installing-manually

Below cargo, above manual

Reply:
You should consider putting installation instructions in the main README so everyone can see it.
It's hard to find it like this.

Reply:
I have a local branch where I am reformatting a bunch of the readme stuff, I want to have as few changes as possible for when I merge it in. The changes I requested will make my merges easier.